Mathlib/Tactic/LinearCombination.lean

```diff
@@ -137,7 +137,7 @@ def elabLinearCombination
     match ← expandLinearCombo ty e with
     | .const c => `(Eq.refl $c)
     | .proof p => pure p
-  let norm := norm?.getD (Unhygienic.run `(tactic| ring1))
+  let norm := norm?.getD (Unhygienic.run <| withRef (← getRef) `(tactic| ring1))
```

Comment:
This puts the error message on the whole tactic, not just the `linear_combination`, right?

If so, presumably Kyle's suggestion [on Zulip](https://leanprover.zulipchat.com/#narrow/stream/270676-lean4/topic/Propagating.20errors.20through.20Unhygienic.2Erun/near/464484715) would resolve it.